client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -317,7 +317,7 @@ public void run() {
                     getChannel().writeAndFlush(new WireCommands.KeepAlive()).addListener(ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE);
                 }
             } catch (Exception e) {
-                log.warn("Keep alive failed, killing connection {} due to {}", connectionName, e.getMessage());
+                log.warn("Failed to send KeepAlive to {}. Closing this connection.", connectionName, e);
```

Comment:
I don't think this works. As far as I am aware, you have to choose between having parameters that are passed as varargs and providing an exception

Reply:
It does work, I have added a couple of snippets below.